parity-util-mem/Cargo.toml

```diff
@@ -19,9 +19,9 @@ parity-util-mem-derive = { path = "derive", version = "0.1" }
 impl-trait-for-tuples = "0.1.3"
 
 smallvec = { version = "1.0.0", optional = true }
-ethereum-types = { version = "0.8.0", optional = true, path = "../ethereum-types" }
+ethereum-types = { version = "0.8.1", optional = true, path = "../ethereum-types" }
 parking_lot = { version = "0.10.0", optional = true }
-primitive-types = { version = "0.6", path = "../primitive-types", default-features = false, optional = true }
+primitive-types = { version = "0.7", path = "../primitive-types", default-features = false, optional = true }
```

Comment:
The bad news is it means a breaking change for parity-util-mem, which in turns is a breaking change for `kvdb-` crates.
The good news is it's just a version bump, no dig deal :)